comp/core/workloadmeta/collectors/internal/containerd/image.go

```diff
@@ -253,7 +256,7 @@ func (c *collector) notifyEventForImage(ctx context.Context, namespace string, i
 	// Build initial workloadmeta.ContainerImageMetadata from manifest and image
 	manifest, err := images.Manifest(ctxWithNamespace, img.ContentStore(), img.Target(), img.Platform())
```

Comment:
when it is possible that `images.Manifest()` return an error?

Reply:
The manifest is the mono-arch manifest (wraps config and compressed layers). In most cases it should be there but I assume that it can be garbage collected at any time.